app.go

```diff
@@ -864,13 +865,33 @@ func (app *App) Hooks() *Hooks {
 	return app.hooks
 }
 
+// TestConfig is a struct holding Test settings
+type TestConfig struct {
```

Comment:
How about

```go
// TestConfig holds the settings for a test.
type TestConfig struct {
    // Timeout defines the maximum duration a test can run before timing out.
    // Default: time.Second
    Timeout time.Duration

    // FailOnTimeout specifies whether the test should return a timeout error
    // if the HTTP response exceeds the Timeout duration.
    // Default: true
    FailOnTimeout bool
}
```

Reply:
`FailOnTimeout` sounds nicer, I will update the code and documentation.